tests/Admin/BaseBlockAdminTest.php

```diff
@@ -42,6 +42,9 @@ public function testSettingAsEditedOnPreBatchDeleteAction()
         $blockAdmin->preBatchAction('delete', $query, $idx, true);
     }
 
+    /**
+     * @group legacy
```

Comment:
If this test is now considered "legacy", shouldn't a "NEXT_MAJOR" comment explain how to handle it in 4.0?